desktop/apps/artwork/components/commercial/templates/index.jade

```diff
@@ -1,5 +1,5 @@
 .artwork-commercial( class='js-artwork-commercial' )
-  form.artwork-commercial__form
+  form.artwork-commercial__form( style= !artwork.is_for_sale ? "padding-top: 0" : "" )
```

Comment:
hmm, not sure if this is right, I think the condition should be `!artwork.is_inquireable`